F2/genotyping.md

```diff
@@ -0,0 +1,142 @@
+<h1 style="text-align:center"><span style="color:#246CAA; font-size:1.5em">Genotyping</span></h1>
+
+Genotyping is characterisation of bacteria based on genetic content, this enables differentiation of bacteria beyond the species/subspecies level. In this module we are focusing on sequenced-based genotyping. The characteristics of typing methods vary in their discriminatory power, and the method needs to be appropriate to address the question e.g. outbreak detection or long-term surveillance. Genotyping using whole-genome sequence data can be conducted in many different ways. In this module we will focus on easy to use, web-based tools. We will build upon your knowledge and introduce command line based tools in the next module.
+
+---
+# Species identification
+
+Determining the species is a crucial first step in your analysis. Species identification using the 16S rRNA gene sequence has been a common method, as 16S rRNA genes are ubiquitous in bacteria and are highly conserved. However, the 16S rRNA represents a very small fraction of the genome and comparing the whole genome sequence data to a reference database of known bacterial species can be more informative than 16S typing. To reduce the computational effort, the reads or contigs can be divided into small pieces, termed _k_-mers. A _k_-mer is a contiguous sequence of _k_ bases, where _k_ can be any positive integer so _7_-mer is a sequence that is 7 bases long. Tools can break the sequence of your isolate of unknown bacterial species into k-mers and compare these k-mers with a curated database of _k_-mers from known bacterial species. Genome sequences that share a high percentage similarity share a high percentage of _k_-mers. In the practical section of this module we will be using the species assignment tool within Pathogenwatch called [Speciator](https://cgps.gitbook.io/pathogenwatch/technical-descriptions/species-assignment/speciator).
```

Comment:
please italicise the 'k' of k-mer where not already done; there are two cases in the sentence begining "Tools can break the sequence..."